homeassistant/components/matter/light.py

```diff
@@ -351,11 +357,21 @@ def _update_from_device(self) -> None:
             self._attr_brightness = self._get_brightness()
 
 
+class ColorControlCapabilities(IntFlag):
```

Comment:
please do not reinvent the wheel. Either create some constants with the upstream int values or just copy the upstream bitmap here so we can have a drop in-replacement once we update the sdk to a newer release.